File: eRxWeb/SPA/Model/Common/grid-pagination.model.ts
export class GridPaginationModel {
    HasNext: boolean;
    HasPrevious: boolean;
    ToTalPageCount: number;
    constructor(
        public PageSize: number,
        public CurrentPageNo: number,
        public TotalNumberOfRecord: number
    ) {
        this.ToTalPageCount = Math.ceil(this.TotalNumberOfRecord / this.PageSize);
        this.HasNext = (this.CurrentPageNo+1 )<this.ToTalPageCount;
        this.HasPrevious = this.CurrentPageNo > 0;
        
    }
}

File: eRxWeb/ServerModel/GridPaginationModel.cs
namespace eRxWeb.ServerModel
{
    public class GridPaginationModel
    {
        public GridPaginationModel()
        {

            CurrentPageNo = 0;
            TotalNumberOfRecord = 0;
        }
        public int PageSize { get; set; }
        public int CurrentPageNo { get; set; }
        public int TotalNumberOfRecord { get; set; }
    }
}